x-pack/legacy/plugins/file_upload/server/kibana_server_services.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+let dataClient;
+
+export const setElasticsearchClientServices = elasticsearch => {
+  ({ dataClient } = elasticsearch);
+};
+export const getDataClient = () => dataClient;
+
+let internalRepository;
+export const setInternalRepository = createInternalRepository => {
+  internalRepository = createInternalRepository();
+};
+export const getInternalRepository = () => internalRepository;
```

Comment:
Any reason to have this here? It appears only used by telemetry so maybe we could just pass it into `registerFileUploadUsageCollector`? I prefer to limit what code has access to the internal repository as much as possible.

Reply:
`updateTelemetry` also calls `getTelemetry` so I'd have to pass it through routes too. It just seemed like a lot of passing of the same service. Would you prefer it if I moved that code over to `telemetry.ts` and made the getter local (i.e.- not exported)?

Reply:
I see, I missed that. This works then 👍